### ISSUE TITLE
[Question]: unable to understand the results

### ISSUE THREAD

**Issue by suruchi-25** [Valdes-Tresanco-MS/gmx_MMPBSA#518]
### My Question is...

i have obtained the .dat and .csv file bt not understanding how to compile the results?

**Comment by Valdes-Tresanco-MS** [comment]
Please, check [this page](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/output/)

**Comment by suruchi-25** [comment]
I have obtained this result but not getting the results same as it. can you
please help

**Comment by Valdes-Tresanco-MS** [comment]
Please, attach the result via GitHub since responses via email did not include the attachment

**Comment by suruchi-25** [comment]
[LIG2_FINAL_RESULTS_MMPBSA.csv](https://github.com/user-attachments/files/16228134/LIG2_FINAL_RESULTS_MMPBSA.csv)
here i am attaching the file.. please give the solution

**Comment by suruchi-25** [comment]
Please help me out with this issue. Waiting for your reply

**Comment by Valdes-Tresanco-MS** [comment]
Well, you can get a summary result in the *.dat file, which is explained in the documentation. 
In this file you share, as you can see, the first line (1A) is the method used, in this case, `GENERALIZED BORN`.
If you check the [introduction in the documentation](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/introduction/), Figure 1, you can see the thermodynamic cycle. So, in the file first, the Complex Energy Terms have one value for every energy term per frame. After that, you can find the receptor, ligand, and delta energy terms, with the same structure as the complex. The energy that permits to prediction if the interaction is favorable or not, is the delta since this value is the rest of the complex minus the sum of its components.
Note that the *.dat file contains the average for all the computed frames.
Please, review the [introduction](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/introduction/) to get close to the algorithms used in gmx_MMPBSA

**Comment by suruchi-25** [comment]
we need to calculate it manually?

**Comment by Valdes-Tresanco-MS** [comment]
No. This is the default [output file](https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/output/) which is always generated and it is a summary of the calculation. With this file, you can see what happeng in the calculation. All the terms are represented by the average, SD, etc. The csv files (they are optional, and must be defined as `-eo file_name`) contains the information of all energy terms per frames that you can visualize in the gmx_MMPBSA_ana.

**Comment by suruchi-25** [comment]
Not able to understand from the tutorial video available on the main site, how to analyze from the gmx_MMPBSA_ana. Is any other tutorial video is availabe of using the ana programm

**Comment by suruchi-25** [comment]
I am really thankful for your input. but can you help me out with the
analysis. how to use the gmx_MMPBSA_ana.
